app/scripts/controllers/permissions/snaps/snap-permissions.test.js

```diff
@@ -8,6 +8,14 @@ import {
   buildSnapRestrictedMethodSpecifications,
 } from './snap-permissions';
 
+// Temporarily replace the snaps packages with the Flask versions.
+jest.mock('@metamask/snaps-controllers', () =>
+  jest.requireActual('@metamask/snaps-controllers-flask'),
+);
+jest.mock('@metamask/rpc-methods', () =>
+  jest.requireActual('@metamask/rpc-methods-flask'),
+);
```

Comment:
Why does this not work otherwise? Does this not break in production?

Reply:
No, it is code-fenced out in production but not in unit tests